Code/GraphMol/RWMol.cpp

```diff
@@ -22,6 +22,102 @@
 
 namespace RDKit {
 
+namespace {
+void insertStereoGroups(RWMol &mol, const ROMol &other,
+                        const std::vector<unsigned int> &newAtomIds,
+                        const std::vector<unsigned int> &newBondIds) {
+  std::vector<RDKit::Atom *> abs_atoms;
+  std::vector<RDKit::Bond *> abs_bonds;
+  std::vector<RDKit::StereoGroup> new_groups;
+  new_groups.reserve(mol.getStereoGroups().size());
+  for (const auto &sg : mol.getStereoGroups()) {
+    if (sg.getGroupType() == RDKit::StereoGroupType::STEREO_ABSOLUTE) {
+      abs_atoms = sg.getAtoms();
```

Comment:
This is going to fail if there is more than one ABS stereo group.
It's easy enough for this to happen:
```
In [12]: m = Chem.MolFromSmiles('C[C@H](O)[C@@H](C)F |a:1|')

In [13]: m = Chem.RWMol(m)

In [14]: sgs = m.GetStereoGroups()

In [15]: sgs.append(Chem.CreateStereoGroup(Chem.StereoGroupType.STEREO_ABSOLUTE,m,[3]))

In [16]: m.SetStereoGroups(list(sgs))

In [17]: m.Debug()
Atoms:
	0 6 C chg: 0  deg: 1 exp: 1 imp: 3 hyb: 4 arom?: 0 chi: 0
	1 6 C chg: 0  deg: 3 exp: 4 imp: 0 hyb: 4 arom?: 0 chi: 2
	2 8 O chg: 0  deg: 1 exp: 1 imp: 1 hyb: 4 arom?: 0 chi: 0
	3 6 C chg: 0  deg: 3 exp: 4 imp: 0 hyb: 4 arom?: 0 chi: 1
	4 6 C chg: 0  deg: 1 exp: 1 imp: 3 hyb: 4 arom?: 0 chi: 0
	5 9 F chg: 0  deg: 1 exp: 1 imp: 0 hyb: 4 arom?: 0 chi: 0
Bonds:
	0 0->1 order: 1 conj?: 0 aromatic?: 0
	1 1->2 order: 1 conj?: 0 aromatic?: 0
	2 1->3 order: 1 conj?: 0 aromatic?: 0
	3 3->4 order: 1 conj?: 0 aromatic?: 0
	4 3->5 order: 1 conj?: 0 aromatic?: 0
Stereo Groups:
	0 ABS rId: 0 wRd: 0 atoms: { 1 }
	1 ABS rId: 0 wRd: 0 atoms: { 3 }
```

Or just a CTAB like this:
```

     RDKit          2D

  0  0  0  0  0  0  0  0  0  0999 V3000
M  V30 BEGIN CTAB
M  V30 COUNTS 6 5 0 0 0
M  V30 BEGIN ATOM
M  V30 1 C 0.000000 0.000000 0.000000 0
M  V30 2 C 1.299038 0.750000 0.000000 0
M  V30 3 O 1.299038 2.250000 0.000000 0
M  V30 4 C 2.598076 -0.000000 0.000000 0
M  V30 5 C 3.897114 0.750000 0.000000 0
M  V30 6 F 2.598076 -1.500000 0.000000 0
M  V30 END ATOM
M  V30 BEGIN BOND
M  V30 1 1 2 1 CFG=3
M  V30 2 1 2 3
M  V30 3 1 2 4
M  V30 4 1 4 5 CFG=1
M  V30 5 1 4 6
M  V30 END BOND
M  V30 BEGIN COLLECTION
M  V30 MDLV30/STEABS ATOMS=(1 2)
M  V30 MDLV30/STEABS ATOMS=(1 4)
M  V30 END COLLECTION
M  V30 END CTAB
M  END
```

Reply:
Well, there should only be on ABS group on a mol, according to the spec, if we get 2 on the same mol, then something bad happened. Should I add a check for that?

Reply:
How about just collapsing them into one - by extending `abs_atoms` instead of replacing it - if/when you hit "extra" ABS groups?

Reply:
That sounds good to me.